### ISSUE TITLE
Use lastUpdate variable

### ISSUE THREAD

**Issue by MTRNord** [MTRNord/ls-vertretungsplan-desktop#18]
Use lastUpdate Variable to do auto refresh

**Comment by johan12345** [comment]
`lastUpdate` only shows the last time where the server refreshed the schedule from the school website (which is approximately once per minute). I'm not sure if that is what you need.

**Comment by MTRNord** [comment]
Ah I thaught that it shows when there was the last time something different :/

**Comment by johan12345** [comment]
No. There is the `lastChange` field that shows the "Stand" from the schedule (but that's not necessarily the same time when it was uploaded to the website and downloaded by my server).

**Comment by MTRNord** [comment]
Ok than that is the better way to go. Means I can check if that (`lastChange` ) changed from the previous request?

**Comment by johan12345** [comment]
You can, but I can't guarantee that there is no change on the schedule when `lastChange` did not change (because the school might be able to edit the schedule without updating the displayed "Stand"). Why don't you just compare the whole schedule?

**Comment by MTRNord** [comment]
Wanted to keep it fast :) It is a little faster to compare 1 field vs multiple fields. But I will try both

**Comment by johan12345** [comment]
Well, with today's computing power you probably won't notice 😉
